### PR TITLE
allow enterprise to import e2e test scripts

### PR DESCRIPTION
**Pull request by ilackarms**
makes test scripts extendable for the enterprise repo

also fixes an internal extender interface

### CODE REVIEW

File: test/e2e/env.go [hunk 1]
```diff
@@ -223,20 +223,27 @@ func StartEnv(ctx context.Context) Env {
...
 	cmd.Stderr = GinkgoWriter
+	cmd.Env = os.Environ()
 
 	err = cmd.Run()
 	if err != nil {
 		dumpState()
 	}
 	Expect(err).NotTo(HaveOccurred())
 
+	err = os.Chdir(filepath.Join(originalWd))
```
Comment (harveyxia) [inline test/e2e/env.go:244]: Do we need `filepath.Join` here?

**Review by harveyxia — APPROVED** [review]

**Review by joekelley — APPROVED** [review]

**Pull request merged**